packages/shared/lib/services/connection.service.integration.test.ts

```diff
@@ -66,4 +67,103 @@ describe('Connection service integration tests', () => {
             expect(updatedMetadata).toEqual({ ...initialMetadata, ...newMetadata });
         });
     });
+
+    describe('listConnections', () => {
+        const googleConfigKey = Math.random().toString(36).substring(7);
+        const notionConfigKey = Math.random().toString(36).substring(7);
```

Comment:
I tried to make tests fully independent by seeding the env and account on each tests, it's debatable but avoid this stuff

Reply:
Sure sure

Reply:
I actually think this is worth a discussion as well, because updating these tests to be isolated would be inconsistent with the rest of the file here and makes it more difficult to read. Feels like a discussion around testing philosophy is in order.

Reply:
same answer as above, those are old tests, them being not great is not a reason to not do better ☺️ 
tbf here using `const { env } = await seeders.seedAccountEnvAndUser();` instead of `env` would solve everything (I think), not a huge diff